source/testing/acceptance.md

```diff
@@ -8,39 +8,47 @@ ember g acceptance-test login
 This generates this file:
 
 ```tests/acceptance/login-test.js
-import { test } from 'qunit';
-import moduleForAcceptance from 'my-app/tests/helpers/module-for-acceptance';
+import { module, test } from 'qunit';
+import { visit } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
 
-moduleForAcceptance('Acceptance | login');
+module('Acceptance | login', function(hooks) {
+  setupApplicationTest(hooks);
 
-test('visiting /login', function(assert) {
-  visit('/login');
-
-  andThen(function() {
+  test('visiting /login', async function(assert) {
+    await visit('/login');
     assert.equal(currentURL(), '/login');
   });
 });
 ```
 
-`moduleForAcceptance` deals with application setup and teardown. The last few lines, within
-the function `test`, contain an example test.
+`module` allows you to scope your tests: Any test setup that is done inside of this scope, will
+apply to all test cases contained in this module. You can read more about module specific setup and
+teardown in the section [Setup and teardown](Acceptance#setup-and-teardown).
+Scoping your tests with `module` also allows you to execute your tests independently from other tests.
+For example, to only run your tests from your `login` module, run `ember t -m 'Acceptance | login'`.
+`setupApplicationTest` deals with application setup and teardown.
+The last few lines, within the function `test`, contain an example test.
 
 Almost every test has a pattern of visiting a route, interacting with the page
 (using the helpers), and checking for expected changes in the DOM.
 
 For example:
 
 ```tests/acceptance/new-post-appears-first-test.js
-import { test } from 'qunit';
-import moduleForAcceptance from 'my-app/tests/helpers/module-for-acceptance';
-
-moduleForAcceptance('Acceptance | posts');
-
-test('should add new post', function(assert) {
-  visit('/posts/new');
-  fillIn('input.title', 'My new post');
-  click('button.submit');
-  andThen(() => assert.equal(find('ul.posts li:first').text(), 'My new post'));
+import { module, test } from 'qunit';
+import { click, fillIn, visit } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+
+module('Acceptance | posts', function(hooks) {
+  setupApplicationTest(hooks);
+
+  test('should add new post', async function(assert) {
+    await visit('/posts/new');
+    await fillIn('input.title', 'My new post');
+    await click('button.submit');
+    assert.equal(this.element.querySelector('ul.posts li:first').textContent, 'My new post');
```

Comment:
Just **begs** for `qunit-dom` 😝 .  I'll try to tackle that (or convince @Turbo87 to take a stab) in a future RFC...

Reply:
Yes, I definitely agree - let's update the examples accordingly as soon as we use `qunit-dom` by default as well

Reply:
I'm curious as to why we don't use `find`, `findWithAssert` and `findAll` a la `ember-cli-native-helpers` instead of `qunit-dom`. They could have been put right into `@ember/test-helpers`.

Reply:
I'm interested in the answer to @Gaurav0 question here as well. @jessica-jordan / @rwjblue do you know if the long-term thinking is to shift away from `find`? Or is this just alternate syntax to using `find()`? Will one of them be preferable over another with the long-term changes to components? What we teach here is likely to be what folks start using who are new to testing, so would love to point them in the best direction possible ...

Reply:
`find` and `findAll` both exist now from `@ember/test-helpers`, but I still believe that we should primarily teach `this.element.querySelector`. The goal of the recent testing refactors is to reduce “magic” by making it very clear what is going on. `find` is yet another possible piece of “magic”, whereas `element.querySelector` is “just” normal DOM API’s...